### DISCUSSION TITLE
Projects & Tasks

### DISCUSSION THREAD

**Discussion by AaronHuston** [outline/outline#4845] [Self Hosting]
Hello! Is it possible to add task creation and tracking? We thought about moving from Notion, but noticed that Outline only has a wiki

**Comment by Rumcajsev** [comment]
There is currently a tasklist functionality, but it doesn't track who has marked it as checked and when it was done (at least it's not visible from document viewer or editors point).
Personally I'd treat this as a good addition - however full projects/tasks handling (trello-like) I think is a bit too much for the scope of this tool.

**Comment by ovizii** [comment]
An option would be to be able to embed this feature i.e. from trello?
